src/dbg/commands/cmd-gui.cpp

```diff
@@ -8,6 +8,36 @@
 #include "value.h"
 #include "variable.h"
 
+bool cbShowThreadId(int argc, char* argv[])

```

Comment:
This should be merged with the existing command: https://github.com/x64dbg/x64dbg/blob/development/src/dbg/commands/cmd-thread-control.cpp#L49